core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -160,7 +160,7 @@ protected[controller] class RestAPIVersion(apipath: String, apiversion: String)(
             sendCorsHeaders {
                 (pathEndOrSingleSlash & get) {
                     complete(OK, info)
-                } ~ authenticate(basicauth) {
+                } ~ (authenticate(basicauth) | authenticate(certificateAuth)) {
```

Comment:
i am still concerned about this route change in the absence of a guard that couple it to the deployment.
if i deploy without the certificate checking in nginx, then i can bypass the basic auth and gain access as any other space by just guessing the subject/namespace.